packages/dds/tree/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -931,6 +932,52 @@ describeForBothConfigs("SequenceField - Sandwich composing", (config) => {
 			assertChangesetsEqual(sandwichParts1to6, []);
 		}),
 	);
+	it.skip("[move, move, modify, move] ↷ [del]", () =>
+		withConfig(() => {
+			const [mo1, mi1] = Mark.move(1, brand(1));
+			const move1 = tagChange([mi1, mo1], tag1);
+			const [mo2, mi2] = Mark.move(1, brand(2));
+			const move2 = tagChange([mi2, mo2], tag2);
+			const mod = tagChange([Mark.modify(TestChange.mint([], 1))], tag3);
+			const [mo3, mi3] = Mark.move(1, brand(3));
+			const move3 = tagChange([mi3, mo3], tag4);
+			const del = tagChange([Mark.delete(1, brand(0))], tag0);
+			const return1 = tagRollbackInverse(invert(move1), tag5, move1.revision);
+			const return2 = tagRollbackInverse(invert(move2), tag6, move2.revision);
+			const unMod = tagRollbackInverse(invert(mod), tag7, mod.revision);
+			const return3 = tagRollbackInverse(invert(move3), tag8, move3.revision);
+			const move1Rebased = rebaseTagged(move1, del);
+			const changes = [
+				return3,
+				unMod,
+				return2,
+				return1,
+				del,
+				move1Rebased,
+				move2,
+				mod,
+				move3,
+			];
```

Comment:
Not sure how readable this series of comments would be for anyone but you or Alex, but that's probably fine. It reads more like "cliff notes" than an explanation of the scenario.

Reply:
Yeah, it's definitely geared towards someone with intimate knowledge of sequence field rebasing and composition.
These comments should only stay as long as the bug isn't fixed.